testscript/testscript.go

```diff
@@ -528,7 +528,7 @@ func (ts *TestScript) condition(cond string) (bool, error) {
 // abbrev abbreviates the actual work directory in the string s to the literal string "$WORK".
 func (ts *TestScript) abbrev(s string) string {
 	s = strings.Replace(s, ts.workdir, "$WORK", -1)
-	if *testWork {
+	if *testWork || ts.params.TestWork {
```

Comment:
oops. I suspect that the `testWork` flag was an unintentional leftover here. I'd kinda like to remove it, but probably best not.